holopy/scattering/theory/scatteringtheory.py

```diff
@@ -42,7 +45,8 @@ class ScatteringTheory(HoloPyObject):
     implementing a _calc_field(self, scatterer, schema) function that returns a
     VectorGrid electric field.
     """
-    def _calc_field(self, scatterer, schema, medium_wavevec, medium_index, illum_polarization):
+    
+    def _calc_field(self, scatterer, schema):
```

Comment:
We should talk about this, I thought we had reached a different conclusion about how we are passing in the metadata than it looks like you have done here. I am not sure you are not right, but I would like to chat about it before making this change.